### ISSUE TITLE
Password not editable in Mautic Credentials

### ISSUE THREAD

**Issue by deepDiverPaul** [n8n-io/n8n#224]
**Describe the bug**
When creating credentials for Mautic, you cannot enter a password.

**To Reproduce**
1. New Credential
2. Mautic API
3. Enter Credentials

**Expected behavior**
You should be able to fill in a password

**Environment (please complete the following information):**
 - OS: Debian 9
 - n8n Version 0.47.0
 - Node.js Version 11.15.0

**Additional context**
packages/nodes-base/credentials/MauticApi.credentials.ts line 26
maybe change `type: 'password' as NodePropertyTypes,` to
```
type: 'string' as NodePropertyTypes,
typeOptions: {
    password: true,
},
```

**Comment by janober** [comment]
Thanks a lot! Yes exactly, I did mess that up.

However, confused why TypeScript and so my IDE did not complain as "password" is no valid value for NodePropertyTypes...

Anyway, it got fixed exactly as you proposed. Will release a new version later.

**Comment by janober** [comment]
Just released n8n@0.48.0 with the fix.
